src/test_pages/BasePage.ts

```diff
@@ -43,4 +43,13 @@ export abstract class BasePage {
       await this.clickOnElement(optionsListElement.locator(`//li[text()='${optionToSelect}']`))
     })
   }
+
+  protected async getAllOptions_Dropbox(dropElement: Locator) {
```

Comment:
```suggestion
  protected async selectAllOptinos(dropElement: Locator) {
```
Would be a better name, don't you think?